t/unit/conftest.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 # we have to import the pytest plugin fixtures here,
-# in case user did not do the `python setup.py develop` yet,
+# in case user did not yet `pip install ".[develop]"`,
```

Comment:
Doesn't it make better sense with "yet" at end of the sentence, as the before?

Reply:
It felt like a dangling participle when I read it.  (But I would bet dollars to doughnuts that it is not a participle. 😃 )